### PR TITLE
Stanet conversion

### PR DESCRIPTION
**Pull request by SimonRubenDrauz**

### CODE REVIEW

File: pandapipes/converter/stanet/data_cleaning.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+
+def remove_connections_zero_length(stored_data, cutoff_length=1e-1, save_path=None,
+                                   write_header=False, save_excel=False):
+    connections = stored_data["house_connections"].copy()
+    used_connections = np.isin(connections.STANETID, stored_data["house_pipes"].CLIENTID) \
+        | np.isin(connections.STANETID, stored_data["house_pipes"].CLIENT2ID)
+    unused_ids = connections.STANETID.loc[~used_connections].values
+
+    cons = connection_pipe_section_table(stored_data, True, False)
```
Comment (SimonRubenDrauz) [inline pandapipes/converter/stanet/data_cleaning.py:44]: the function has four inputs. One is missing. @dlohmeier, do you know, which one?

File: pandapipes/converter/stanet/stanet2pandapipes.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+logger = logging.getLogger(__name__)
+
+
+# TODO: Some things are left to do for this converter:
+#         - in case of house connection calculation in STANET, the connections have results VMA and
+#           VMB. These results can be used for comparison, but also for sorting of connections on a
+#           main pipe --> especially if connection nodes on the main pipe have the same coordinates.
+#         - some TODOs in table_creation
+#         - maybe it will be necessary to remove deleted data from the STANET tables, otherwise they
+#           might be inserted into the pandapipes net erroneously
```
Comment (SimonRubenDrauz) [inline pandapipes/converter/stanet/stanet2pandapipes.py:32]: despite the opne todos I think we should merge the PR. Maybe add some infos. What do yo think, @dlohmeier

File: pandapipes/converter/stanet/table_creation.py [hunk 3]
```diff
@@ -0,0 +1,1161 @@
...
+        stanet_nr=pipes.RECNO.values, stanet_id=pipes.STANETID.values,
+        geodata=pipe_sections.section_geo.values, v_stanet=pipes.VM.values,
+        stanet_system=CLIENT_TYPES_OF_PIPES[MAIN_PIPE_TYPE],
+        stanet_active=pipes.ISACTIVE.values.astype(np.bool_),
+        stanet_valid=~pipes.CALCBAD.values.astype(np.bool_),
+        **add_info
+    )
+
+
+def create_heat_exchanger_sta(net, stored_data, connection_table, index_mapping, add_layers):
```
Comment (SimonRubenDrauz) [inline pandapipes/converter/stanet/table_creation.py:543]: Why sta, @dlohmeier ?

File: pandapipes/converter/stanet/data_cleaning.py (hunk 1 above)
Comment (dlohmeier) [reply pandapipes/converter/stanet/data_cleaning.py]: I corrected this.

File: pandapipes/converter/stanet/stanet2pandapipes.py (hunk 2 above)
Comment (dlohmeier) [reply pandapipes/converter/stanet/stanet2pandapipes.py:32]: For the first version, this should be fine.

**Review by dlohmeier — APPROVED** [review]
I think we have done it. Almost all Todos from #390 have been implemented, the first version should be ready to go. Adaptions to be implemented later...

**Review by jkisse — APPROVED** [review]

**Pull request merged**